### PR TITLE
 Better progressive playback controls of bottom pages

### PR DESCRIPTION
**Pull request by xlinbsd**
The volume slider is important to conserve at most width for audio playing.
The new value of hiding seems best balanced.

Only tested on last Firefox on Linux. Other tests for a better compromise are welcome.

### CODE REVIEW

**Comment by xlinbsd** [comment]
Tested with latests Linux Chromium and Electron OK too.

**Comment by sparky8251** [comment]
Do be aware, there are 2 other places in code that the volume slider exists. Its also in videoosd.js and remotecontrol.js.

Might not have to touch the videoosd.js one, but the remotecontrol.js is actually used in the second music playback screen. The one where you have your queued playlist along the bottom.

**Comment by xlinbsd** [comment]
@sparky8251 OK Seen. Thanks

**Comment by xlinbsd** [comment]
Hello, is the new ordering please you ?

**Comment by xlinbsd** [comment]
I included my modifications of second pull since it is in the same objective to improve control bars

**Comment by sparky8251** [comment]
Have any before and after pictures?

**Comment by xlinbsd** [comment]
@sparky8251 I'll do it

**Comment by xlinbsd** [comment]
after

**Comment by xlinbsd** [comment]
Before.

before

**Comment by xlinbsd** [comment]
Tested on latest iOS Safari, Lineage 16, macos 13's Safari, Linux Firefox && Linux Chromium

**Comment by sparky8251** [comment]
It's looking good. Let us know when you feel its "complete".

**Comment by xlinbsd** [comment]
I think it is complete for now. Need other users testing (in 10.2.3 ?) the new control style to be sure it is better for majority of us, no ?
I'll stay close if you think I can help a bit, I'm on the Riot room. I do know more than just some CSS coding.

File: src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css [hunk 1]
```diff
@@ -119,38 +119,75 @@
...
         display: none !important;
     }
+
 }
 
-@media all and (max-width: 50em) {
+@media all and (max-width: 61em) {
 
-    .nowPlayingBarCenter {
+    .nowPlayingBarCenter .stopButton {
```
Comment (vitorsemeano) [inline src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css]: Why did you add the stopButton class here? this is inside the div with the class nowPlayingBarCenter

File: src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css [hunk 2]
```diff
@@ -119,38 +119,75 @@
...
 
-    .toggleRepeatButton {
-        display: none;
+}
+
+
+@media all and (max-width: 55em) {
+
+    .nowPlayingBarCenter .nextTrackButton, .nowPlayingBarCenter .previousTrackButton, .nowPlayingBarCenter .playPauseButton {
+        display: none !important;
```
Comment (vitorsemeano) [inline src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css]: did you try without the !important? can't see to find more classes that use this sub classes with a display none important.

Also, nowPlayingBarCenter is parent of some of those classes. Try removing all sub classes of this one.

File: src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
 
         html += '<button is="paper-icon-button-light" class="muteButton mediaButton"><i class="md-icon">&#xE050;</i></button>';
 
-        html += '<div class="sliderContainer nowPlayingBarVolumeSliderContainer hide" style="width:100px;vertical-align:middle;display:inline-flex;">';
+        html += '<div class="sliderContainer nowPlayingBarVolumeSliderContainer hide" style="width:150px;vertical-align:middle;display:inline-flex;">';
```
Comment (vitorsemeano) [inline src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.js]: I really don't like !important. could that be because of this inline style? display inline-flex? perhaps moving this rules to nowplayingbar.css will ease the !important? i could be asking too much already

File: src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css (hunk 2 above)
Comment (vitorsemeano) [reply src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css]: forget that last part, you are applying the rules to the sub classes, not the parent one.

File: src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css (hunk 1 above)
Comment (vitorsemeano) [reply src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css]: also forget this one, you are only applying the rule to stopButton, makes sense.

File: src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css (hunk 2 above)
Comment (xlinbsd) [reply src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.css]: I never myself use !important. That is a residue of the old code. I'll try without, it is a good idea.

File: src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.js (hunk 3 above)
Comment (xlinbsd) [reply src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.js]: I will check if !important is really important.

Comment (xlinbsd) [reply src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.js]: After some tests, `!important` is needed in this state of code. I will check why

**Comment by xlinbsd** [comment]
Sorry for the last pushes, I was editing directly in github since I decided to do it right with git commands from my pc. I missed some editings while trying to do things right.
